src/components/Authors/Authors.tsx

```diff
@@ -0,0 +1,98 @@
+import React, {Fragment} from 'react';
+import block from 'bem-cn-lite';
+import {Author} from '../../models';
+
+import './Authors.scss';
+
+const b = block('authors');
+
+const MAX_NUMBER_DISPLAYED_AUTHORS = 3;
```

Comment:
```suggestion
const MAX_DISPLAYED_AUTHORS = 3;
```
MAX кажется и так указывает, что это number